### PR TITLE
Adds bin/setup and Procfile

### PR DESCRIPTION
**Pull request by amoose**
Includes bin/setup and Procfile from my original [PR](https://github.com/fugacious/fugacious/commit/25762a7d905d1eca4aeda08c232847d47cf3865c)

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -t 5:5 -p ${PORT:-3000} -e ${RACK_ENV:-development}
```
Comment (jmcarp) [inline Procfile]: We also have this configuration in `config/puma.rb`, using it in `bin/cf-start`.

Comment (amoose) [inline Procfile]: Procfile is required for foreman and Heroku; does this collide with config/puma.rb ?

Comment (jmcarp) [inline Procfile]: What I'm suggesting is something like:

```
web: bundle exec puma -C config/puma.rb
```

...as in `bin/cf-start`.

Comment (amoose) [inline Procfile]: AH -- yes, I think that is the correct approach. I will update!

**Comment by amoose** [comment]
Updated Procfile @jmcarp and tested on Heroku 🚀

**Pull request merged**